workflow-bridge-lambda/src/index.ts

```diff
@@ -7,19 +7,43 @@ const WORKFLOW_DATASTORE_API_URL = `http://${getEnvironmentVariableOrThrow(
   "workflowDnsName"
 )}/api`;
 
-exports.handler = async (event: {
-  arguments?: { composerId?: string; ids?: string[]; searchText?: string };
-}) => {
+const fields = [
+  "id",
+  "title",
+  "composerId",
+  "headline",
+  "trashed",
+  "path",
+].join(",");
+
+interface StubsResponseBody {
+  data: {
+    content: { [status: string]: PinboardData[] };
+  };
+}
+
+interface Arguments {
+  composerId?: string;
+  ids?: string[];
+  searchText?: string;
+  paths?: string[];
+}
+
+exports.handler = async (event: { arguments?: Arguments }) => {
   if (event.arguments?.composerId) {
     return await getPinboardById("content")(event.arguments.composerId);
   }
   if (event.arguments?.ids) {
+    // TODO do this in single call using new endpoint added in https://github.com/guardian/workflow/pull/1119
```

Comment:
workflow#1119 is merged, does this still need doing?

Reply:
it does need doing, but will do it in a follow-up